### ISSUE TITLE
Visualize the final prediction

### ISSUE THREAD

**Issue by Sundragon1993** [shjung13/Standardized-max-logits#5]
Dear Sanghun Jung,

Thank you so much for your amazing works, I was wondering if you could provide the source which helps visualize the b) Unexpected detected and c) final prediction as you stated in the paper?
Thank you and looking forward to hearing from you!

**Comment by shjung13** [comment]
Hello, sorry for the late reply and thank you for your interest in our work!

We currently don't have visualization codes that we used for figures.

For (b) Unexpected detected in the Teaser figure, we visualize them using heatmap, and for the figure (c), we add CYAN color for the OOD class and modified the ```cityscapes_labels.py``` code to visualize the segmentation map.

Hope this helps you and please let me know if you have any questions :)

Thank you.
